examples/credential_issuer_metadata_jwt_vc_json.json

```diff
@@ -14,7 +14,7 @@
             "locale": "fr-FR"
         }
     ],
-    "credentials_supported": {
+    "credential_configurations_supported": {
```

Comment:
I think `credential_configurations` is enough (as this is a credential offer already).